assets/js/common/SuseManagerConfig/SuseManagerConfig.jsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { defaultTo, noop } from 'lodash';
+
+import Button from '@common/Button';
+
+import CertificateUploadDate from './CertifiedUploadDate';
+
+const renderPassword = (username, certUploadDate) =>
+  username && certUploadDate ? '•••••' : '.....';
+
+function SuseManagerConfig({
+  url = 'https://',
+  username,
+  certUploadDate,
+  onEditClick = noop,
+}) {
+  return (
+    <div className="container max-w-7xl mx-auto p-4 sm:p-6 lg:p-8 bg-white dark:bg-gray-800 rounded-lg">
+      <div>
+        <h2 className="text-2xl font-bold inline-block">SUSE Manager Config</h2>
+        <span className="w-32 inline-block float-right">
+          <Button type="primary-white" size="small" onClick={onEditClick}>
+            Edit Settings
+          </Button>
+        </span>
+      </div>
+      <p className="mt-3 mb-3 text-gray-500">
+        SUSE Manager lets you discover software vulnerabilities on nodes
```

Comment:
Slightly different from the design where we have `SUSE Manager enables software updates for hosts.`
No preferences, just aiming to get alignment with @jagabomb

Reply:
I think this also works for me. The best person check this product language with would be @abravosuse.